index.html

```diff
@@ -22,18 +61,24 @@
     <form id="calc" onSubmit="return calc();">
       <fieldset form="calc">
         <legend>Input</legend>
-        <label for="mac">MAC address</label>
-        <input name="mac" id="mac" placeholder="de:ad:be:ef:00:00"/>
-        <button type="submit">Calculate</button>
+        <span class="input">
+          <label for="mac">MAC address</label>
+          <input name="mac" id="mac" placeholder="de:ad:be:ef:00:00"/>
+          <button type="submit">Calculate</button>
+        </span>
       </fieldset>
       <fieldset form="calc">
         <legend>Output</legend>
-        <label for="mesh">Mesh IP</label>
-        <input name="mesh" id="mesh" disabled="disabled"/>
-        <button id="copy-mesh" type="button">copy</button>
-        <label for="wlan">WLAN IP</label>
-        <input name="wlan" id="wlan" disabled="disabled"/>
-        <button id="copy-wlan" type="button">copy</button>
+        <span class="output">
```

Comment:
span for display inline, changed to block < 400px. Is it better to use div and apply display: inline?